ci/scripts/common.sh

```diff
@@ -97,7 +97,6 @@ function filter_stack_trace() {
   touch tmp
   cat "$1" \
   | sed -E '/  [1-9][0-9]+:/d' \
-  | sed -E '/  [3-9]+:/d' \
```

Comment:
Interesting to know this 🥵

Reply:
Yeah.. I had to add function in because the recovery tests had too much logs. But turns out it's overly conservative. So just add some improvement in this PR.